builder/azure/arm/step_delete_resource_group_test.go

```diff
@@ -49,35 +49,32 @@ func TestStepDeleteResourceGroupShouldPassIfDeletePasses(t *testing.T) {
 	}
 }
 
-func TestStepDeleteResourceGroupShouldTakeStepArgumentsFromStateBag(t *testing.T) {
-	var actualResourceGroupName string
-
+func TestStepDeleteResourceGroupShouldDeleteStateBagArmResourceGroupCreated(t *testing.T) {
```

Comment:
For complex test cases I usually write the description in a comment instead, which is a bit easier to read / add context if needed.